src/api/services/github/GithubApiClient.ts

```diff
@@ -0,0 +1,85 @@
+import { IFetchClient } from "../../utils/interfaces";
+import { IGithubApiClient } from "./interfaces";
+import {
+  GithubMember,
+  GithubTeam,
+  GithubTree,
+  GithubTreeResponse,
+} from "./types";
+import { handleError } from "../../common";
+
+export class GithubApiClient implements IGithubApiClient {
+  private static readonly BASE_URL = "https://api.github.com";
+  private static readonly API_VERSION = "application/vnd.github.v3+json";
```

Comment:
버전은 이 문자열의 한 부분이니 `MEDIA_TYPE`이라고 부르는 게 좀 더 정확하지 않을까요?

```suggestion
  private static readonly MEDIA_TYPE = "application/vnd.github.v3+json";
```

Reply:
`application/vnd.github+json`이라고만 해도 충분할텐데, 버전을 고정하시는 이유가 혹시 있으신가요?

Reply:
@DaleSeo 아니요, 별다른 이유는 없고 검색중 확인한 값을 그대로 사용했습니다.
조금 더 찾아보니 깃헙에서는 버전을 명시하지 않는것을 권장하네요. 수정하겠습니다~!